bundles/model/org.eclipse.smarthome.model.persistence/src/org/eclipse/smarthome/model/persistence/extensions/PersistenceExtensions.java

```diff
@@ -410,6 +411,128 @@ static public DecimalType averageSince(Item item, AbstractInstant timestamp, Str
             return Collections.emptySet();
         }
     }
+    
+	/**
+	 * Query for the last update timestamp of a given <code>item</code>.
+	 * The default persistence service is used.
+	 *
+	 * @param item the item to check for state updates
+	 * @return point in time of the last update or null if none available
+	 */
+	static public Date lastUpdate(Item item) {
```

Comment:
@clinique All other methods use AbstractInstant in their signature and are not using Date. I think it should be the same here, so that it is in line with the rest.